docs/integrations/builtin/core-nodes/n8n-nodes-base.splitinbatches.md

```diff
@@ -8,14 +8,15 @@ contentType: integration
 
 The Loop Over Items node helps you loop through data.
 
-The node saves the original incoming data, and with each iteration, returns a predefined amount of data through the **loop** output.
+The node saves the original incoming data, and with each iteration, returns a predefined amount of data through the **Loop** output.
 
-When the node execution completes, it combines all the data and returns it through the **done** output.
+The Loop Over Items node stops executing after all the incoming items get divided into batches and passed on to the next node in the workflow, so it's not necessary to add an IF node to stop the loop. When the node execution completes, it combines all the data and returns it through the **Done** output. 
```

Comment:
```suggestion
The Loop Over Items node stops executing after all the incoming items get divided into batches and passed on to the next node in the workflow, so it's not necessary to add an IF node to stop the loop. When the node execution completes, it combines all the data and returns it through the **done** output. 
```

Reply:
Matching the case of UI copy exactly (I know it looks weird, but it's 'done' in the UI)